content/pda.md

```diff
@@ -0,0 +1,453 @@
+# PDAs
+
+# Lesson Objectives
+
+_By the end of this lesson, you will be able to:_
+
+- Explain Program Derived Addresses (PDAs)
+- Understand how PDAs are derived
+- Use PDA derivations to map data
+
+# TL;DR
+
+- A Program Derived Address (PDA) is derived from a program ID and an optional list of seeds.
+- PDAs are designed to be controlled by the program they are derived from
+- PDAs provide a deterministic way to find data stored on accounts
+- Programs can sign instruction for PDAs derived from its program ID
+
+## What is a Program Derived Address
+
+Program Derived Addresses (PDAs) are 32 byte strings that look like public keys designed to be controlled by a specific program. A PDA is derived from a program ID and some seeds, but unlike a keypair, a PDA _does not_ have a corresponding private key.
+
+PDAs serve two main functions:
+
+1. Provide a deterministic way to find the address of an account
+2. Allow a program to sign for a PDA (without a private key) in the same way a user may sign with their wallet.
```

Comment:
@ZYJLiu the second point's parenthetical is a little ambiguous.  Do we mean

- a program without a private key
- to sign without a private key
- a PDA without a private key